### PR TITLE
CORS & CSRF

### PR DESCRIPTION
**Pull request by lyssdod**

### CODE REVIEW

**Comment by landswellsong** [comment]
Where do we have CORS?

**Comment by lyssdod** [comment]
In deploy

**Comment by landswellsong** [comment]
Between differing ports?

**Comment by lyssdod** [comment]
Yes

File: promis/promis/settings.py [hunk 1]
```diff
@@ -88,6 +90,16 @@
 
 WSGI_APPLICATION = 'promis.wsgi.application'
 
+# CORS
```
Comment (landswellsong) [inline promis/promis/settings.py]: Make this depend on `DJANGO_DEBUG` variable please.

File: promis/promis/urls.py [hunk 2]
```diff
@@ -17,9 +17,11 @@
 from django.contrib import admin
 from django.conf.urls.i18n import i18n_patterns
 
+from ajax_redirect import ajax_redirect
+from decorator_include import decorator_include
 
 urlpatterns = [
-    url(r'^i18n/', include('django.conf.urls.i18n'))
+    url(r'^i18n/', decorator_include(ajax_redirect, 'django.conf.urls.i18n'))
```
Comment (landswellsong) [inline promis/promis/urls.py]: This requires a comment.

**Comment by landswellsong** [comment]
General comments: 
- Why should we do CORS headers here instead of nginx config?
- Why shouldn't we `Allow-Origin: *`, any downside to that?

**Comment by 0486** [comment]
anti-CORS headers attaching by Django by default.

**Comment by landswellsong** [comment]
Closing this for now, re-open if we decide to go that route.

**Pull request closed without merging**